_data/db.json

```diff
@@ -0,0 +1,39 @@
+{
+  "restaurants": [
+    {
+      "id": 1,
+      "name": "La Petite Brasserie",
+      "description": "Classic French cuisine with a modern twist.",
+      "menu": ["Coq au Vin", "Ratatouille", "Crème Brûlée"],
```

Comment:
Can you add a "tags" key that has an array of values like `"tags: ["italian"]" to these objects?